ports/espressif/boards/mixgo_ce_serial/mpconfigboard.mk

```diff
@@ -20,4 +20,4 @@ CIRCUITPY_MODULE=wroom
 
 FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_Requests
 FROZEN_MPY_DIRS += $(TOP)/frozen/Adafruit_CircuitPython_NeoPixel
-FROZEN_MPY_DIRS += boards/mixgo_ce_udisk/cp_lib/mixgoce_lib
+FROZEN_MPY_DIRS += $(TOP)/frozen/mixgo_circuitpython_lib
```

Comment:
@[dhalbert](https://github.com/dhalbert)，Thank you. Please add another level of directory,`FROZEN_MPY_DIRS += $(TOP)/frozen/mixgo_circuitpython_lib/mixgoce_lib`.because [dahanzimin/circuitpython_lib](https://github.com/dahanzimin/circuitpython_lib.git) is still adding other board libraries

Reply:
@dahanzimin if you want to fix it in your own PR, that is fine. Then I can approve your PR (I cannot approve my own). Right now our build is broken, so that's why I went ahead and made a PR.

> Please add another level of directory,`FROZEN_MPY_DIRS += $(TOP)/frozen/mixgo_circuitpython_lib/mixgoce_lib`.because [dahanzimin/circuitpython_lib](https://github.com/dahanzimin/circuitpython_lib.git) is still adding other board libraries

Are these libraries for other boards? Would you consider breaking these out into separate repos, so that you can maintain them independently? I am not sure what you want visible as the top-level module.

Side note: Thanks for tagging the `circuitpython_lib` release. The frozen library mechanism does require that you use a release tag on the commit to freeze. I was going to mention that but you already did it.

Is this your personal GitHub account? Do you work for MixGo? Have you considered creating a GitHub organization for MixGo? I see https://github.com/MixGO, but I don't think that is you folks. Also I see https://github.com/mixly, but that looks specific for the programming environment.

Reply:
I'm very sorry to bring you trouble. I'm a member of the mixly team, because GitHub is not very friendly in China, and the project is mainly in [gitee](https://gitee.com/mixly2)